upload/inc/languages/english/admin/isango.lang.php

```diff
@@ -29,3 +29,6 @@
 $l['isango_input_mode_max'] = 'Big size buttons with text';
 $l['isango_input_mode_min'] = 'Small buttons with icon only';
 $l['isango_input_mode_pop'] = 'A select dropdown';
+
+$l['isango_uninstall'] = 'Isango - uninstallation';
```

Comment:
Set capital `U` for uninstallation. Remove hyphen (-) to match with MyBB Title Style

Reply:
Done!